proto/side/gmm/tx.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package side.gmm;
+
+option go_package = "github.com/sideprotocol/side/x/gmm/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "side/gmm/pool_params.proto";
+import "side/gmm/pool_asset.proto";
+
+// Msg defines the Msg service.
+service Msg {
+  rpc CreatePool   (MsgCreatePool  ) returns (MsgCreatePoolResponse  );
+  rpc AddLiquidity (MsgAddLiquidity) returns (MsgAddLiquidityResponse);
+  rpc Withdraw     (MsgWithdraw    ) returns (MsgWithdrawResponse    );
+  rpc Swap         (MsgSwap        ) returns (MsgSwapResponse        );
+
+}
+message MsgCreatePool {
+           string     creator   = 1;
+           PoolParams params    = 2;
+  repeated PoolAsset  liquidity = 3 [(gogoproto.nullable) = false];
+}
+
+message MsgCreatePoolResponse {
+  string pool_id = 1;
+}
+
+message MsgAddLiquidity {
+           string                   creator   = 1;
+           string                   pool_id   = 2;
+  repeated cosmos.base.v1beta1.Coin liquidity = 3 [(gogoproto.nullable) = false];
+}
+
+message MsgAddLiquidityResponse {
+  string pool_id = 1;
+}
+
+message MsgWithdraw {
+  string creator = 1;
+  string receiver = 2;
```

Comment:
`creator` and `receiver` are typical `sender`?

Reply:
creator and sender has same meaning. it's just scaffold value with ignite

Reply:
the reason is that `sender` is widely used in msgs of official module. 
and many UIs query messages by keyword:  `msg.sender`. 
if we use `creator` those txs would not be queried.

Reply:
updated